### PR TITLE
Appium setup + simple e2e test

### PR DESCRIPTION
**Pull request by unixvb**

### CODE REVIEW

File: src/components/mnemonic/mnemonic-input/mnemonic-input.tsx [hunk 1]
```diff
@@ -46,6 +47,7 @@ export const MnemonicInput: FC<MnemonicProps> = ({
         onBlur={handleBlur}
         onFocus={() => setIsFocused(true)}
         onChangeText={onChangeText}
+        testID={testID}
```
Comment (andrewkryshtal) [inline src/components/mnemonic/mnemonic-input/mnemonic-input.tsx:50]: do we need here `setTestId()`?

Comment (andrewkryshtal) [reply src/components/mnemonic/mnemonic-input/mnemonic-input.tsx:50]: looks like `react native` `0.64` supports `testId` as well

https://stackoverflow.com/a/68746717/6559770

Comment (unixvb) [reply src/components/mnemonic/mnemonic-input/mnemonic-input.tsx:50]: `setTestId` is applied to `react-native` components (ex `TextInput`, `TouchableOpacity`).  
here I'm just drilling `testID` prop to the native componnets

Comment (unixvb) [reply src/components/mnemonic/mnemonic-input/mnemonic-input.tsx:50]: I have tried settings up `testID` props without `accessibilityLabel` - it does not work

**Review by LendiHop — APPROVED** [review]

**Review by yevhenii-orykhivskyi — APPROVED** [review]

**Pull request merged**